docker/docker-compose-dev.yml

```diff
@@ -43,7 +43,7 @@ x-dashboard-environment: &dashboard-environment
   REDIS_CELERY_DB: 1
   REDIS_CONSTANCE_DB: 2
   SECRET_KEY: ${DASHBOARD_VIEWER_SECRET_KEY}
-  DASHBOARD_VIEWER_ENV: ${INSTALLATION_ENV}
+  DASHBOARD_VIEWER_ENV: development
```

Comment:
@aspedrosa hardcoded?

Reply:
This is the development docker-compose file. Note the file name.